ssl-config-generator/index.html

```diff
@@ -334,7 +334,7 @@
                     }
                     if (data.hstsEnabled == "true") {
                         data.hsts = '\n    # HSTS (ngx_http_headers_module is required) (15768000 seconds = 6 months)' + '\n' +
-                            '    add_header Strict-Transport-Security max-age=15768000;';
+                            '    add_header Strict-Transport-Security "max-age=15768000" always;';
```

Comment:
'always' here is only valid in nginx >= 1.7.5; per the example a couple lines below, this needs to be wrapped in a ```isSemVer(data.serverVersion, ">=1.7.5")``` test to ensure compatibility.